### PR TITLE
Fixes PAI examine bug

### LINKED ISSUE #22206: pAI hologram bugs
Can't examine the pAI holograms.
Pretty sure I saw one pull someone earlier.
AND ALL ACCESS JUST WHYYY
I'll edit it if I see anymore.
12-14-16 6:00 pm EST Bagil

### PR DESCRIPTION
**Pull request by silicons**
Fixes #22206 
examine bugs

### CODE REVIEW

**Comment by Supermichael777** [comment]
still all access

File: code/modules/mob/living/silicon/pai/examine.dm [hunk 1]
```diff
@@ -1,2 +1,3 @@
-/mob/living/silicon/pai/examine() //removed as it was pointless...moved to the pai-card instead.
```
Comment (Supermichael777) [inline code/modules/mob/living/silicon/pai/examine.dm]: just delete this segment and add it into the main file. A three line proc dosen't need a file.

**Comment by kevinz000** [comment]
if only i could do that with the web editor, guess this gets to wait til I get to a working computer
also all access was fixed ages ago.

**Comment by PKPenguin321** [comment]
Did that actually remove those files from the dme

**Comment by RemieRichards** [comment]
@PKPenguin321 no, because then the DME would be in the diff, which it isn't, so he hasn't removed them correctly.

**Comment by AnturK** [comment]
@kevinz000 Please. DME changes.

**Comment by kevinz000** [comment]
fuck

**Comment by kevinz000** [comment]
fixed

**Pull request merged**